src/components/Electricity/ElectricityForm.tsx

```diff
@@ -0,0 +1,140 @@
+import { Button, Select, Card, Label, TextInput } from 'flowbite-react';
+import React, { useState } from 'react';
+import StatesData from '../../AppData/states.json';
+import {
+  CountryCode,
+  ElectricityUnit,
+  UnregisteredElectricityRequest,
+} from '../../schema/electricity.schema';
+
+interface ElectricityFormProps {
+  handleSubmit: (flightData: UnregisteredElectricityRequest) => void;
+}
+const ElectricityForm = ({ handleSubmit }: ElectricityFormProps) => {
+  const [country, setCountry] = useState<CountryCode>('us');
+  const [state, setState] = useState('al');
+  const [electricityUnit, setElectricityUnit] =
+    useState<ElectricityUnit>('mwh');
+  const [electricityValue, setElectricityValue] = useState(0);
+
+  const handleFormSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    handleSubmit({
+      electricity_value: electricityValue,
+      electricity_unit: electricityUnit,
+      country: country,
+      state: state,
+    });
+  };
+
+  const resetForm = () => {
+    setCountry(() => 'us');
+    setState(() => 'al');
+    setElectricityValue(() => 0);
+    setElectricityUnit(() => 'mwh');
+  };
+
+  console.log(
```

Comment:
Should we not check in debugging console logs into dev/main?

Reply:
Nice catch! Updated!